modules/changes-logs/classes/class-changes-handle-wp-database.php

```
@@ -0,0 +1,485 @@
+                // Action Plugin Component.
+                $plugin_file = '';
+
+                if ( isset( $_GET['plugin'] ) ) {
+                    $plugin_file = sanitize_text_field( wp_unslash( $_GET['plugin'] ) );
+                } elseif ( isset( $_GET['checked'] ) && isset( $_GET['checked'][0] ) ) {
+                    $plugin_file = sanitize_text_field( wp_unslash( $_GET['checked'][0] ) );
+                } else {
+
+                    global $wp_current_filter;
+                    if ( isset( $wp_current_filter ) && ! empty( $wp_current_filter ) ) {
+                        foreach ( $wp_current_filter as $key => $value ) {
```

PHPMD (2.15.0)

Avoid unused local variables such as '$key'. (Unused Code Rules)

(UnusedLocalVariable)